src/main/java/seedu/address/model/attendance/AttendanceMark.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.model.attendance;
+
+import seedu.address.model.assignment.Mark;
+
+/**
+ * Represents an attendance mark in the app.
+ * Guarantees: immutable; is valid as declared in {@link #isValid(String)}
+ */
+public class AttendanceMark extends Mark {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "AttendanceMark should be a 0 or a 1";
+
+    /**
+     * Constructs a {@code AttendanceMark}.
+     *
+     * @param mark A valid real number.
+     */
+    public AttendanceMark(String mark) {
+        super(mark);
```

Comment:
Ehh you don't check if mark given was 1 or 0 here? Why not isValid here?

Reply:
I followed the implementation of Mark, and the isValid is tested in ParserUtil

Reply:
Okay!